projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/badge-docs.component.html

```diff
@@ -1,139 +1 @@
-<header class="go-page-title">
```

Comment:
Just curious, where did all this code go? Was it not needed anymore? Seems like a lot of code to just be deleted...

Reply:
its added to app-positioned-badge-docs